src/main/java/gregtech/api/items/gui/ItemUIFactory.java

```diff
@@ -38,6 +38,7 @@ default GTGuiTheme getUITheme() {
         return GTGuiTheme.STANDARD;
     }
 
+    // TODO: change to abstract once MUI2 port is complete
```

Comment:
Are there any other Items that need a MUI2 GUI that can be opened from the hand?